### ISSUE TITLE
Places Manager

### ISSUE THREAD

**Issue by javacado** [happydenn/cordova-plugin-gimbal2#1]
Thanks for this plugin! I just recently upgraded from your 1.0 version to this. However, I am wondering if you are planning on integrating any PlaceManager functionality in the future? I appreciate your work in bringing this plugin to the community.

**Comment by happydenn** [comment]
Yes. The plan is to integrate as much feature from the v2 SDK as possible. But due to the very limited time I have currently I cannot give a specific time frame on this. Glad you find this plugin useful! :)

**Comment by joeflateau** [comment]
@happydenn I'm prepared to open a PR for Places Manager if you'd accept it.
